lua/nvim-tree/actions/tree/modifiers/expand-all.lua

```diff
@@ -47,13 +59,14 @@ local function gen_iterator()
     Iterator.builder(parent.nodes)
       :hidden()
       :applier(function(node)
-        if should_expand(expansion_count, node) then
+        if should_expand(expansion_count, node, populate_node) then
```

Comment:
![20240907_150214](https://github.com/user-attachments/assets/b6a2c443-7c05-41d0-8506-ee56911c1df7)

This appears unused. Can we remove it?